Source/Tests/TrackableEntities.EF.5.Tests/FamilyDbContextTests.cs

```diff
@@ -215,5 +215,21 @@ public void Apply_Changes_Should_Mark_Grandchild_Deleted()
             Assert.Equal(EntityState.Deleted, context.Entry(child).State);
             Assert.Equal(EntityState.Deleted, context.Entry(grandchild).State);
         }
+
+        [Fact]
+        public void Apply_Changes_Should_Mark_ComplexType_Modified()
+        {
+            // Arrange
+            var context = TestsHelper.CreateFamilyDbContext(CreateFamilyDbOptions);
+            var parent = new MockFamily().Parent;              
+            parent.Address.StreetName = "123 Lee Ave.";
+            parent.Address.ModifiedProperties = new[] { nameof(Address.StreetName) };
```

Comment:
What's missing here is to set `TrackingState` to `Modified`.  AFAIK currently TE first checks `TrackingState` before `ModifiedProperties`.  The reason is that the purpose of `ModifiedProperties` is to allow for partial updates.  If there are no modified properties, then a full update us performed if `TrackingState` is `Modified`.  So to stay consistent, I'd like to keep this requirement for complex types as well.